src/main/java/team9/ddang/walk/controller/request/CompleteWalkRequest.java

```diff
@@ -0,0 +1,23 @@
+package team9.ddang.walk.controller.request;
+
+import jakarta.validation.constraints.NotNull;
+import team9.ddang.walk.service.request.CompleteWalkServiceRequest;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+
+@Schema(description = "산책 완료 Request DTO")
+public record CompleteWalkRequest(
+
+        @Schema(description = "산책 총 거리 (미터)", example = "1200")
+        @NotNull(message = "산책 총 거리가 존재해야 합니다.")
+        Integer totalDistance,
+
+        @Schema(description = "산책 총 시간 (초)", example = "1800")
+        @NotNull(message = "산책 총 시간이 존재해야 합니다.")
+        Long totalWalkTime
```

Comment:
검증 잘 하고 계시지만, 값의 범위에 대한 검증도 추가된다면 좋을 것 같아요!
```suggestion
        @Schema(description = "산책 총 거리 (미터)", example = "1200")
        @NotNull(message = "산책 총 거리가 존재해야 합니다.")
        @Positive(message = "산책 총 거리는 0보다 커야 합니다.")
        Integer totalDistance,

        @Schema(description = "산책 총 시간 (초)", example = "1800")
        @NotNull(message = "산책 총 시간이 존재해야 합니다.")
        @Positive(message = "산책 총 시간은 0보다 커야 합니다.")
        Long totalWalkTime
```

Reply:
네! 다시 올려야겠네요